virttest/utils_windows/drive.py

```diff
@@ -86,3 +86,23 @@ def shrink_volume(session, vol_id, size):
     shrink_cmd += 'echo shrink desired=%s >> {0} ' % size
     shrink_cmd += '&& diskpart /s {0}'
     session.cmd(shrink_cmd.format(script_path))
+
+
+def get_disk_property_by_serial_number(session, serial_number, property_name):
```

Comment:
It look like this function support multi properties, that is good. It match the real requirement. My test include 3 usages:
1.get_disk_property_by_serial_number(session, serial_number,"Name") # single 
2.get_disk_property_by_serial_number(session, wrong_serial_number,"Name")
3.get_disk_property_by_serial_number(session, serial_number,"Index,Name") # multi

for the usage1 the str return
for the usage3 the dict returned
for the usage2 what should be rerturn,so far, it always re-do the operation.

Reply:
Update the codes by your comments, please check.